packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -177,6 +177,15 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
       setActivePane('none');
     } else {
       setActivePane('chat');
+      // timeout is required to give the window time to render the sendbox so we have something to send focus to.
+      // TODO: look into another way to select this/ adjust composite to select this better.
+      const chatFocusTimeout = setInterval(() => {
+        const textBox = document.querySelector(`[id="sendbox"]`) as HTMLTextAreaElement;
+        if (textBox !== null) {
+          textBox.focus();
+          clearInterval(chatFocusTimeout);
```

Comment:
You also want an overall timeout after which you `clearInterval`. If, for some reason, the chat pane isn't shown at all, this will keep polling forever.

Reply:
will do!